### DISCUSSION TITLE
Installing types?

### DISCUSSION THREAD

**Discussion by benbrunyee** [dicebear/dicebear#144] [Q&A] [answered]
How can I install types for this package. The expected @types/dicebear/avatars doesn't work :/

**Comment by FlorianKoerner** [comment, marked as answer]
The package "[@dicebear/avatars](https://www.npmjs.com/package/@dicebear/avatars)" already contains the types. You can start right away. No types from "@types" necessary. 🥳

**Reply by benbrunyee** [reply]
Oh didn't notice that. Awesome!
